clang-tidy-required-packages.yaml

```diff
@@ -0,0 +1,223 @@
+##### common #####
```

Comment:
About this file as a whole, what about removing the commented out packages?

This way the file will be cleaner. Right now it is hard to see the "not commented out" packages.

We can just keep adding new packages as they come through.

Reply:
This way, the prefixing job won't affect it that much.

Reply:
Agreed :+1:

Reply:
Ok thanks, I'll approve once the commented out packages are removed from this file :+1:

Reply:
@yhisaki 
Could you update clang-tidy-required-packages.yaml file?

Reply:
@yhisaki 
I have create a new, minimum `.clang-tidy-ci` configuration, so please use it and list up the packages which pass the minimum check.
https://github.com/autowarefoundation/autoware/pull/5465